tests/acceptance/stepDefinitions/sharingContext.js

```diff
@@ -196,6 +197,25 @@ Given(
   }
 )
 
+Given('the administrator has enabled exclude groups from sharing', function () {
+  return runOcc(
+    [
+      'config:app:set core shareapi_exclude_groups --value=yes'
+    ]
+  )
+})
+
+Given('the administrator has excluded group {string} from sharing', function (group) {
+  return runOcc(
+    [
+      'config:app:set',
+      'core',
+      'shareapi_exclude_groups_list',
+      '--value=["' + group + '"]'
```

Comment:
When there is only one group to exclude from sharing then this is fine. But when we want to implement steps like
```
Given the administrator has excluded group "grp1" from sharing
And the administrator has ecluded group "grp2" from sharing
```
This won't work. 
Please make this work for that kind of cases too.